etc/helm/examples/enterprise-dev.yaml

```diff
@@ -9,3 +9,6 @@ enterpriseServer:
     tag: local
 pachd:
   enabled: false
+pgbouncer:
+  enabled: true
```

Comment:
This should be the default

Reply:
I believe this should be false now, as we're not using PG bouncer for enterprise?

Reply:
Yeah, we have to pass credentials for PG_BOUNCER though since the enterprise server is now equivalent to a full pachd.  They can also point to postgres, but they do have to be set.